Content.Server/Administration/BwoinkSystem.cs

```diff
@@ -28,7 +29,8 @@ protected override void OnBwoinkTextMessage(BwoinkTextMessage message, EntitySes
 
             // TODO: Sanitize text?
             // Confirm that this person is actually allowed to send a message here.
-            if ((senderSession.UserId != message.ChannelId) && (_adminManager.GetAdminData(senderSession) == null))
+            var senderPersonalChannel = senderSession.UserId == message.ChannelId;
+            if (senderPersonalChannel && (_adminManager.GetAdminData(senderSession) == null))
```

Comment:
mfw i try to ahelp but i can't type

Reply:
...did I seriously manage to screw up this condition
oh dear goodness
nice catch, thank you